tools/build_file_generator/BuildFileGenerator.java

```diff
@@ -168,12 +167,12 @@ private JavaTest getJavaTest(JavaClass javaClass) {
 
       String target;
       if (thirdPartyTargets.isEmpty()) {
-        // it isn't third party dep
-        target = getInternalDep(imprt, getWholeProjectProtoFiles());
+        // It isn't third party dep.
+        target = getInternalDep(importProto, getWholeProjectProtoFiles());
       } else if (thirdPartyTargets.size() == 1) {
         target = thirdPartyTargets.get(0).getTarget();
       } else {
-        // we choose the smallest
+        // If the class exists in several third party deps we choose the smallest one.
```

Comment:
I'd add: "since the other ones probably contain it"

Reply:
Done.